building_sim_plugins/building_ignition_plugins/src/slotcar.cpp

```diff
@@ -201,9 +239,53 @@ std::vector<Eigen::Vector3d> SlotcarPlugin::get_obstacle_positions(
   return obstacle_positions;
 }
 
+void SlotcarPlugin::charge_state_cb(const ignition::msgs::Selection& msg)
+{
+  dataPtr->charge_state_cb(msg.name(), msg.selected());
+}
```

Comment:
This change is related to the battery charging PR right? If there is too many of those changes might be worth it to move them to a separate PR, just to make tracking changes easier.

Reply:
I think the function just got reordered, which is why Git seems to show it as a new addition. All the battery charging changes have been merged into master already.